recipe/meta.yaml

```
@@ -1,41 +1,46 @@
 {% set name = "pyuca" %}
 {% set version = "1.2" %}
-{% set hash_type = "sha256" %}
-{% set hash = "8a382fe74627f08c0d18908c0713ca4a20aad5385f077579e56208beea2893b2" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  {{ hash_type }}: {{ hash }}
+  sha256: 8a382fe74627f08c0d18908c0713ca4a20aad5385f077579e56208beea2893b2
 
 build:
-  noarch: python
   number: 1
-  script: python -m pip install --no-deps --ignore-installed .
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed --no-build-isolation .
 
 requirements:
   host:
     - pip
     - python
     - setuptools
-
+    - wheel
   run:
     - python
 
 test:
   imports:
     - pyuca
+  requires:
+    - pip
+  commands:
+    - pip check
 
 about:
   home: https://pypi.org/project/pyuca
   license: MIT
```

```suggestion
  license: MIT AND Unicode-3.0
```
As highlighted by the Fedora package: https://src.fedoraproject.org/rpms/python-pyuca/blob/rawhide/f/python-pyuca.spec#_9. And if we really want to be accurate, we would need https://src.fedoraproject.org/rpms/python-pyuca/blob/rawhide/f/28.patch I think.

This can also be confirmed via https://www.unicode.org/copyright.html (see 3.c)

---

ALso, see https://github.com/jtauber/pyuca/pull/28, and more importantly https://github.com/jtauber/pyuca/pull/28#issuecomment-1975218185.

---

The licensing issue is more complicated, IMHO. The Spyder's maintainer said (on Jan 5, https://github.com/jtauber/pyuca/issues/27#issue-2066412999):
`However, without clarification, pyuca cannot be included in official Fedora repositories and that might be the end of Spyder, a much loved IDE, in Fedora.`
The PR hasn't been approved since then and we don't know the answer of `pyuca`'s maintainer. The same Spyder's maintainer created the [patch](https://src.fedoraproject.org/rpms/python-pyuca/blob/rawhide/f/28.patch). It's not a common way.
Should we also ask the `pyuca`'s maintainer to solve the licensing issue before patching the package?

---

I trust myself, Penguinpee and Fedora. https://www.unicode.org/copyright.html is very clear, see clause 3.c. Also, https://github.com/jtauber/pyuca/pull/28#issuecomment-1975218185 and https://src.fedoraproject.org/rpms/python-pyuca/blob/rawhide/f/28.patch mean "fedora approved" AFAIK.

---

I reread Unicode's terms of use and https://www.unicode.org/Public/UCA/10.0.0/ReadMe.txt. So, yeah, `MIT AND Unicode-3.0` will be fine with an added patch.